src/common/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/cmake/Common.cmake)
 
 add_subdirectory(redis_module)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -g")
```

Comment:
Any reason not to compile with debug info all the time?

Reply:
size of the executables is the only one that comes to mind, which we don't care about that much